crates/strategies/opensea-sudo-arb/contracts/README.md

```diff
@@ -0,0 +1,262 @@
+address deployment = SudoOpenseaArbCompiled.deploy();
+
+// returns the initialization code.
+bytes memory initcode = SudoOpenseaArbCompiled.initcode;
+```
+
+## Documentation
+
+Developer documentation is provided for all definitions including description of macro directives and panic cases.
+
+The main contract may be found [here](./src/huff/SudoOpenseaArb.huff).
+
+## Verification
+
+To assert the `huffc` compiler output matches the bytecode, the following script may be run.
+
+> Notice: This requires `ffi` to be enabled to execute. The command may be found [here](./scripts/VerifyHuff.s.sol).
+
+```bash
+# runs the script with ffi enabled.
+# `huffc` is required for verification.
+forge script ./scripts/VerifyHuff.s.sol --ffi
+```
+
+## ABI Specification
+
+The Huff contract maintains feature parity with the Solidity implementation.
+
+### `executeArb(BasicOrderParameters calldata, uint256, address)`
+
+Custom data types:
+
+```solidity
+struct BasicOrderParameters {
+    address considerationToken;                     // 0x24
+    uint256 considerationIdentifier;                // 0x44
+    uint256 considerationAmount;                    // 0x64
+    address payable offerer;                        // 0x84
+    address zone;                                   // 0xa4
+    address offerToken;                             // 0xc4
+    uint256 offerIdentifier;                        // 0xe4
+    uint256 offerAmount;                            // 0x104
+    BasicOrderType basicOrderType;                  // 0x124
+    uint256 startTime;                              // 0x144
+    uint256 endTime;                                // 0x164
+    bytes32 zoneHash;                               // 0x184
+    uint256 salt;                                   // 0x1a4
+    bytes32 offererConduitKey;                      // 0x1c4
+    bytes32 fulfillerConduitKey;                    // 0x1e4
+    uint256 totalOriginalAdditionalRecipients;      // 0x204
+    AdditionalRecipient[] additionalRecipients;     // 0x224
+    bytes signature;                                // 0x244
+    // Total length, excluding dynamic array data: 0x264 (580)
+}
+
+enum BasicOrderType {
+    // -- snip --
+}
+
+struct AdditionalRecipient {
+    uint256 amount;
+    address payable recipient;
+}
+```
+
+Calldata layout:
+
+```
+bop = basic_order_params
+
+selector
+    . bop_ptr
+    . payment_value
+    . sudo_pool
+    . bop_consideration_token
+    . bop_consideration_identifier
+    . bop_consideration_amount
+    . bop_offerer
+    . bop_zone
+    . bop_offer_token
+    . bop_offer_identifier
+    . bop_offer_amount
+    . bop_start_time
+    . bop_end_time
+    . bop_zone_hash
+    . bop_salt
+    . bop_offerer_conduit_key
+    . bop_fulfuller_conduit_key
+    . bop_total_original_additional_recipients
+    . bop_additional_recipients_ptr
+    . bop_signatures_ptr
+    . bop_additional_recipients_len
+    . (bop_additional_recipients_amount . bop_additional_recipients_recipient)+
```

Comment:
nit: is the formatting on this line intentional? was a bit confused about how to interpret, the point is that you can have more than one pair of `bop_additional_recipients_amount and bop_additional_recipients_recipient` right?

Reply:
right, `+` means one or more. would it be better to annotate this or is there a better way to describe it?